lib/rules/no-unpublished-import.js

```diff
@@ -42,13 +43,15 @@ module.exports = {
             options.ignoreTypeImport === void 0
                 ? false
                 : options.ignoreTypeImport
+        const ignorePrivate =
+            options.ignorePrivate === void 0 ? true : options.ignorePrivate
```

Comment:
This would be better as:
```suggestion
        const ignorePrivate = options.ignorePrivate ?? true
```

Reply:
Wasn't sure whether I could use `??` given the line above did it exactly like that.

Converted all 3 default value fallbacks to the suggested syntax.